src/pages/AdminConsole.js

```diff
@@ -18,7 +18,6 @@ class AdminConsole extends Component {
       showModal: false,
       showAddUserModal: false,
       dulUnreviewedCases: 0,
-      darUnreviewedCases: 0,
```

Comment:
I removed darUnReviewedCases from this code because it was presenting a no-unused-vars when compiling.
It was necessary for the pre MDV code. Is it okay for me to include these changes here?

Reply:
Yes, that's fine, good catch.